### DISCUSSION TITLE
Custom Fan Bug

### DISCUSSION THREAD

**Discussion by MTK6891** [seerge/g-helper#2144] [General]
Hello, I have discovered a bug. Specifically, it is as follows: apply a custom fan mode, check the option to start automatically, and after restarting, the fan speed remains at 1900RPM, even if the CPU temperature reaches 80 degrees. If you switch modes at this time, it will return to normal.😢

**Comment by seerge** [comment]
@MTK6891 hello,

1. You write in a wrong place (this is discussions, not issues).
2. Fans are always controlled by BIOS (even in case of custom fan curves). https://github.com/seerge/g-helper/wiki/FAQ#how-does-g-helper-control-my-fan-speeds

**Reply by MTK6891** [reply]
Sorry, my feedback was sent to the wrong place
